### ISSUE TITLE
WDL graph not created when generating report

### ISSUE THREAD

**Issue by Mauritz8** [franciscoBSalgueiro/en-croissant#443]
### Describe the bug

When generating a computer analysis report for a game, the WDL (win-draw-loss) graph is not created. I was able to reproduce it using both stockfish and leela chess zero.

![image](https://github.com/user-attachments/assets/71dff7c1-4a06-40fc-8482-e6af4f9c6d04)

### Reproduction

1. Import game from pgn
2. Go to report tab
3. Click generate report
4. Choose Stockfish 17 in the popup
5. Wait for report to finish
6. Select the WDL graph
7. It is not generated as expected

### Platform and versions

Ubuntu 24.04

**Comment by JWilterdink** [comment]
I encountered this issue as well - it wound up being that in advanced settings for my engine, "UCI_ShowWDL" was not checked. I enabled this setting and I'm getting the WDL graph now. Maybe check for yours and see if that is the issue?

Regardless, might be good to give some sort of error message on this screen if that checkbox is not checked, or even an option to enable it right then and there. I just randomly stumbled across it, could have taken a while to find if I didn't get lucky.
